src/main/java/com/coveros/selenified/utilities/Reporter.java

```diff
@@ -79,6 +78,7 @@
 
 
     // the image width for reporting
+    public static final String[] ENABLED_LOGS = new String[]{LogType.BROWSER, LogType.CLIENT, LogType.DRIVER, LogType.PERFORMANCE, LogType.PROFILER, LogType.SERVER};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this member "protected". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS2386)